pkg/cmd/upgrade/upgrade_cli.go

```diff
@@ -19,7 +16,9 @@ import (
 
 var (
 	upgradeCLILong = templates.LongDesc(`
-		Upgrades the Jenkins X command line tools if there is a newer release
+		Upgrades the Jenkins X command line tools if there is a new version available in the version stream
+
+		For more information on Version Streams see: [https://jenkins-x.io/architecture/version-stream/](https://jenkins-x.io/architecture/version-stream/)
```

Comment:
Probably want to say that the reference to the version stream is stored in team settings somewhere?